src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -188,6 +196,18 @@ public void resumeCommissioning() {
     resumeCommissioning(deviceControllerPtr);
   }
 
+  /**
+   * 
+   * The following fields on the ControllerParams object must be populated:
+   * - ipk
+   * - rootCertificate
+   * - intermediateCertificate
+   * - operationalCertificate
+   * - adminSubject
+   * 
+   * @param params
+   * @return
+   */
```

Comment:
This doc is insufficient and the name `setNOCChain` is still quite confusing

Reply:
added more complete comments